go.mod

```diff
@@ -33,7 +40,7 @@ require (
 	github.com/libp2p/go-libp2p-host v3.0.15+incompatible
 	github.com/libp2p/go-libp2p-interface-connmgr v0.0.21 // indirect
 	github.com/libp2p/go-libp2p-interface-pnet v3.0.0+incompatible // indirect
-	github.com/libp2p/go-libp2p-kad-dht v4.4.12+incompatible
+	github.com/libp2p/go-libp2p-kad-dht v4.4.13-0.20190129000003-bebd753a1d86+incompatible
```

Comment:
This hard dependency on a specific commit is not good. Can't we refer to master HEAD?

Reply:
This is addressed now, but I think the original concern is ill-conceived.